sirepo/job_supervisor.py

```diff
@@ -691,7 +700,15 @@ def _req_is_valid(self, req):
                         await self.run_dir_acquire(o)
                     await o.prepare_send()
                     o.send()
-                    return await o.reply_get()
+                    r =  await o.reply_get()
+                    # POSIT: All non runSimulation jobs that could run into
```

Comment:
runSimulation or here are the only places we need to check for runDirNotFound.

Reply:
Can you phrase what your saying a different way? I'm not sure what you're saying.

Reply:
The POSIT is that we know all the places that runDirNotFound needs to be caught. There are two: runSimulation and this location. "All non runSimulation" implies that runSimulation also needs to catch runDirNotFound. I thought it should be more explicit.

Probably being too pedantic.

Reply:
Thanks. I rephrased it. Thoughts?